src/persist-client/src/lib.rs

```diff
@@ -312,8 +314,24 @@ impl PersistConfig {
             writer_lease_duration: 60 * Duration::from_secs(60),
             reader_lease_duration: Self::DEFAULT_READ_LEASE_DURATION,
             critical_downgrade_interval: Duration::from_secs(30),
+            // TODO: This doesn't work with the process orchestrator. Instead,
+            // separate --log-prefix into --service-name and --enable-log-prefix
+            // options, where the first is always provided and the second is
+            // conditionally enabled by the process orchestrator.
```

Comment:
related, it seems like it'd be nice to source this var from one of the params (probably interpreted by `clap` at the process entrypoint). but, maybe in the follow up PR that makes this work with the process orchestrator

Reply:
yeah, I 100% don't want to mess with any of this myself in this PR :D